applets/cmv12k/spi_test.py

```diff
@@ -0,0 +1,52 @@
+# set up and demonstrate SPI connection to CMV12k control pins
+
+from nmigen import *
+from naps import *
+
+class Top(Elaboratable):
+    def __init__(self):
+        self.sensor_reset_n = ControlSignal(name='sensor_reset', reset=1)
+
+    def elaborate(self, platform: BetaPlatform):
+        m = Module()
+
+        platform.ps7.fck_domain(requested_frequency=100e6)
+
+        sensor = platform.request("sensor")
+        platform.ps7.fck_domain(250e6, "sensor_clk")
+        m.d.comb += sensor.lvds_clk.eq(ClockSignal("sensor_clk"))
+        m.d.comb += sensor.reset.eq(~self.sensor_reset_n)
```

Comment:
shouldnt the inversion be rather handled by the platform?

Reply:
i converted this to non-inverted logic, unlike the other applets, but it makes more sense that way